From 6d8a5ad2ad124354598f3778a5dc947ba54a8817 Mon Sep 17 00:00:00 2001
From: Gabe Black <gabe.black@gmail.com>
Date: Fri, 15 Apr 2022 16:44:20 -0700
Subject: [PATCH 565/757] stdlib: Add missing ()s in get_supported_isas.

A function was used directly instead of being called when interating
through ISA names.

Change-Id: I9dc53259a766a68070e0b79bb55f721a93fcdeed
Reviewed-on: https://gem5-review.googlesource.com/c/public/gem5/+/58929
Reviewed-by: Jason Lowe-Power <power.jg@gmail.com>
Maintainer: Jason Lowe-Power <power.jg@gmail.com>
Tested-by: kokoro <noreply+kokoro@google.com>
---
 src/python/gem5/runtime.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/python/gem5/runtime.py b/src/python/gem5/runtime.py
index 108dfc4..0984f6f 100644
--- a/src/python/gem5/runtime.py
+++ b/src/python/gem5/runtime.py
@@ -48,7 +48,7 @@ def get_supported_isas() -> Set[ISA]:
     if "TARGET_ISA" in buildEnv.keys():
         supported_isas.add(get_isa_from_str(buildEnv["TARGET_ISA"]))
     else:
-        for key in get_isas_str_set:
+        for key in get_isas_str_set():
             if buildEnv[f"USE_{key.upper()}_ISA"]:
                 supported_isas.add(get_isa_from_str(key))
     return supported_isas
-- 
1.8.3.1

